tests/Get-DbaDatabase.Tests.ps1

```diff
@@ -35,6 +35,32 @@ Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
             $results.IsAccessible | Should Be $true
         }
     }
+
+}
+
+Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
+    BeforeAll {
+
+        $random = Get-Random
+        $dbname1 = "dbatoolsci_Backup_$random"
+        $dbname2 = "dbatoolsci_NoBackup_$random"
+        New-DbaDatabase -SqlInstance $script:instance1 -name $dbname1 , $dbname2
```

Comment:
Maybe add `$null = ` to suppress the output of `New-DbaDatabase` (and change `-name ` to `-Name`?)

Reply:
Change makes sense to me, should i open a new PR for it?

Reply:
Just if you have time...